### DISCUSSION TITLE
Export Folder  with subfolder and single PDF files

### DISCUSSION THREAD

**Discussion by ih8snow** [l1xnan/obsidian-better-export-pdf#186] [General]
Hi,
thanks for the great plugin.
I wondered if it would be possible to offer an alternative to existing one-page PDF export.

My idea is to export a folder from my vault with the given subfolder structure and each .md in these folders is exported as single PDF file.

Ist that possible to do?

**Comment by l1xnan** [comment]
Currently, export by folder is supported, see https://github.com/l1xnan/obsidian-better-export-pdf#quick-export . It's just slower if you have a lot of files.

**Reply by ih8snow** [reply]
Hi,
Yes, I understand that but this creates one large PDF file.
My idea is to get single PDF files per each md file in a folder, so basically a folder with 5 .md files comes back as a folder with 5 PDF files.
Example:
main folder plus 4 subfolder should return the same folder structure but with PDF files instead of  .md files.

**Reply by l1xnan** [reply]
I got it. I'll add supporting this feature to my to-do list.

**Reply by l1xnan** [reply]
@ih8snow I in the 1.9.0 version of the initial support, you can batch export all the notes under the folder as a PDF file, the use of the process of any problem, welcome to feedback!
